api/src/test/java/com/epam/pipeline/controller/security/PermissionControllerTest.java

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright 2017-2020 EPAM Systems, Inc. (https://www.epam.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.epam.pipeline.controller.security;
+
+import com.epam.pipeline.controller.vo.EntityPermissionVO;
+import com.epam.pipeline.controller.vo.PermissionGrantVO;
+import com.epam.pipeline.entity.security.acl.AclClass;
+import com.epam.pipeline.entity.security.acl.AclSecuredEntry;
+import com.epam.pipeline.manager.security.AclPermissionApiService;
+import com.epam.pipeline.test.web.AbstractControllerTest;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.web.servlet.MvcResult;
+
+import static com.epam.pipeline.test.creator.CommonCreatorConstants.ID;
+import static com.epam.pipeline.test.creator.CommonCreatorConstants.TEST_STRING;
+import static com.epam.pipeline.test.creator.security.PermissionCreatorUtils.ACL_SECURED_ENTRY_TYPE;
+import static com.epam.pipeline.test.creator.security.PermissionCreatorUtils.ENTITY_WITH_PERMISSION_VO_TYPE;
+import static com.epam.pipeline.test.creator.security.PermissionCreatorUtils.getAclSecuredEntry;
+import static com.epam.pipeline.test.creator.security.PermissionCreatorUtils.getEntityPermissionVO;
+import static com.epam.pipeline.test.creator.security.PermissionCreatorUtils.getPermissionGrantVO;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.verify;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+
+public class PermissionControllerTest extends AbstractControllerTest {
+
+    private static final String GRANT_URL = SERVLET_PATH + "/grant";
+    private static final String ALL_PERMISSIONS_URL = GRANT_URL + "/all";
+    private static final String OWNER_URL = GRANT_URL + "/owner";
+    private static final String PERMISSIONS_URL = SERVLET_PATH + "/permissions";
+    private static final String ID_AS_STRING = String.valueOf(ID);
+    private static final String ID_PARAM = "id";
+    private static final String ACL_CLASS_PARAM = "aclClass";
+    private static final AclClass ATTACHMENT = AclClass.ATTACHMENT;
```

Comment:
Could you please fix this test according to your another [PR](https://github.com/epam/cloud-pipeline/pull/1618) ?